templates/secrets/custom-airflow-secret.yaml

```diff
@@ -0,0 +1,22 @@
+################################
```

Comment:
@kushalmalani we shouldn't need this since Commander will take care of creating the secret out-of-band from the helm release cycle. Was this for testing purposes?

Reply:
We would need this if someone is going to use airflow charts in new mode, since with the flag enabled we tell pods to pick this secret file, which wouldnt exists and all pods fail with CreateContainerConfigError